crates/amaru-ledger/src/rules/transaction.rs

```diff
@@ -85,6 +85,9 @@ pub fn execute(
     transaction_witness_set: &MintedWitnessSet<'_>,
     transaction_auxiliary_data: Option<&AuxiliaryData>,
 ) -> Result<(), InvalidTransaction> {
+    // FIXME: this is temporary, to be replaced when we have some state that determines the node's network
+    let network = Network::Testnet;
```

Comment:
`Network` is accessible from the [config](https://github.com/pragma-org/amaru/blob/main/crates/amaru/src/stages/mod.rs#L79). Make sure it is passed correctly from there.

Reply:
Thank you!

Reply:
I guess `network` has to be added to some state for this stage which feels like it should be a different PR.